gobblin-service/src/test/java/org/apache/gobblin/service/modules/core/GitConfigMonitorTest.java

```diff
@@ -128,7 +128,7 @@ public void cleanUp() {
       this.flowCatalog.stopAsync().awaitTerminated();
     }
 
-    cleanUpDir(TEST_DIR);
+    //cleanUpDir(TEST_DIR);
```

Comment:
Why is this removed?

Reply:
I had removed it for debugging purposes. Forgot to uncomment it back.